lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -298,35 +298,26 @@ class SPIRVInstTemplateBase : public SPIRVInstruction {
                                : getValue(Ops[I]);
   }
 
-  // Get the offset of operands.
-  // Some instructions skip literals when returning operands.
-  size_t getOperandOffset() const {
-    if (hasExecScope() && !isGroupOpCode(OpCode) && !isPipeOpCode(OpCode))
-      return 1;
-    return 0;
-  }
-
   // Get operands which are values.
   // Drop execution scope and group operation literals.
   // Return other literals as uint32 constants.
```

Comment:
Should this comment be updated as dropping is removed from the function?

Reply:
I think it can be removed, since we don't drop anything now.